internal/provider/resource_aci_rest_managed.go

```diff
@@ -395,14 +416,21 @@ func splitImportId(importId string) []string {
 		idParts = append(idParts, importId[startIndex:])
 	}
 
-	return idParts
+	return idParts, nil
 
 }
 
 func (r *AciRestManagedResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	tflog.Debug(ctx, "Start import state of resource: aci_rest_managed")
 
-	idParts := splitImportId(req.ID)
+	idParts, err := splitImportId(ctx, req.ID)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Unable to parse import JSON string",
```

Comment:
is this correct when the old split could also error?

Reply:
Done!